tests/unit_tests/models/test-environment.php

```diff
@@ -18,6 +18,90 @@ public function __construct() {
   }
 
   /**
+   * Ensure correct connection info for development environments
+   * Development environment connection info includes git parameters
+   *
+   * @return void
+   *
+   * @vcr site_connection-info
+   */
+  public function testConnectionInfoDev() {
+    logInWithBehatCredentials();
+    $site     = $this->sites->get('behat-tests');
+    $test_env = $site->environments->get('dev');
```

Comment:
I ❤️  that you're even more persnickety than me. I would have called it `$sut_env` or `$subject_env` or something if I had thought about it more. Role more than Identifier is what I would be after in the naming for this testing context.